ingame_ui.py

```diff
@@ -0,0 +1,81 @@
+import arcade
+from pyglet.gl import GL_NEAREST
+
+from entity.player import PlayerInventory
+
+
+class IngameUI:
+    """
+    Used to display in-game interface features like Key count and Current Level.
+    """
+
+    def __init__(self, player_inventory: PlayerInventory):
+        self.player_inv = player_inventory
+        self.inv_sprite = arcade.SpriteList()
+        self.key_sprite = arcade.Sprite("assets/sprites/key.png", 2)
+        self.inv_sprite.append(self.key_sprite)
+        self.colour = int(0x22), int(0x3D), int(0x28)
```

Comment:
don't need to use `int` here, they're already integers

Reply:
True, I was being lazy in converting Hex to Ints. the middle is 3D and it's also not the same base.

Reply:
Do you want this changing? or is my lazy hex conversion fine.

Reply:
wdym, they're ints, just represented as hex. just remove the `int` call and it should work the exact same

Reply:
Oh shit, you rite. Dunno why I thought they would be a Hex obj